api/project.rb

```diff
@@ -28,6 +28,7 @@
 end
 
 command = ARGV.first
-args = ARGV.drop(1)
 
-common.handle_or_die(command, *args)
+args = Hash[ ARGV.drop(1).flat_map{|s| s.scan(/--?([^=\s]+)(?:=(\S+))?/) } ]
+
+common.handle_or_die(command, args)
```

Comment:
Done; using OptionParser for this command. 

At some point we might consider working that into the general common command framework here, but we can do that later.